Models/Index4Response.js

```diff
@@ -44,6 +43,8 @@ export default class Index4Response extends RequestBaseResponse {
         size,
         setting.deform
       );
+      const dirPath = path.dirname(destinationPath);
+      await fs.promises.mkdir(dirPath, { recursive: true });
```

Comment:
Please check that the directory does not exist before creating it.
check `Index4Response._createDirectoryIfNotExist()`